catkin_ws/src/auv_msgs/msg/VisionObject.msg

```diff
@@ -0,0 +1,9 @@
+# Vision Object class
+string label
+#pose info
+float64 x
+float64 y
+float64 z
+float64 theta_z
+#additional data field (for additional info on objects like lane marker second heading)
+float64 extra_field
```

Comment:
it's since different objects have different features, so I just had extra_field there as a way of storing information which is object-specific (second heading for lane marker, which symbol is on the left for gate etc.). I agree it's probably not the cleanest solution but not part of the scope of this issue, if you want to change it we should make a seperate ticket and maybe give it to one of the new members since it'll be a relatively easy change